drf_spectacular/plumbing.py

```diff
@@ -508,6 +508,8 @@ def build_root_object(paths, components, version) -> _SchemaType:
         root['tags'] = settings.TAGS
     if settings.EXTERNAL_DOCS:
         root['externalDocs'] = settings.EXTERNAL_DOCS
+    if webhooks:
+        root['webhooks'] = webhooks
```

Comment:
Should we add support for Redoc's x-webhooks when OAS_VERSION is 3.0.x?

Reply:
let's ignore that for now until someone actually asks for it.